person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age, name: 'Unknwon', parent_permission: true)
+    @id = id
```

Comment:
- [ ] It would be advisable rather depends upon the `user_input` please generate the `id` automatically using `random` library because, in this way your `id` will be always unique and there are the very less chance of the duplicacy in data.